addons/playersuppression/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,13 @@
+class Extended_PreInit_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_FILE(XEH_preInit));
+    };
+};
+
+class Extended_Fired_EventHandlers {
+    class All {
+        class ADDON {
+            clientFired = QUOTE(if ((GVAR(enablePlayerSuppression) && !isDedicated)) then {_this call FUNC(fired)});
```

Comment:
Yes, but you only want to supress players don't you?
`hasInterface` is true on all machines someone is actively playing on. It's false on deicated servers and healess clients. You don't want to supress headless clients or did I miss something?